control_plane/src/storage.rs

```diff
@@ -139,67 +135,52 @@ impl PageServerNode {
                 .collect::<Vec<_>>()
                 .join(",")
         );
-        let mut args = Vec::with_capacity(20);
-
-        args.push("--init");
```

Comment:
Probably you will need to modify this script too
https://github.com/neondatabase/neon/blob/dc102197df0469da544463253e636395d1d33789/.github/ansible/scripts/init_pageserver.sh#L28-L30

Reply:
Thank you, I've considered options and decided to replicate `--init` from safekeeper as discussed above.

Reply:
Added [a test](https://github.com/neondatabase/neon/pull/2272/files#diff-85440eac3bb111914ec6d1e6029a82c37f6dc7f303dda8a33d9c7d56e4ffc8b9R22) for the new `--init` command to replicate this script.